diff -up mpg123-1.33.7/src/compat/compat_dl.c.omv~ mpg123-1.33.7/src/compat/compat_dl.c
--- mpg123-1.33.7/src/compat/compat_dl.c.omv~	2026-04-26 16:34:55.269560891 +0200
+++ mpg123-1.33.7/src/compat/compat_dl.c	2026-04-26 16:35:52.115242859 +0200
@@ -41,6 +41,10 @@
 #include <shlwapi.h>
 #endif
 
+#ifndef HAVE_DLFCN_H
+#define HAVE_DLFCN_H 1
+#endif
+
 #ifdef USE_MODULES
 #  ifdef HAVE_DLFCN_H
 #    include <dlfcn.h>
